src/java/fr/paris/lutece/plugins/forms/web/admin/FormJspBean.java

```diff
@@ -778,9 +778,10 @@ private File getLogoFromRequest( HttpServletRequest request )
     {
         _uploadHandler.addFilesUploadedSynchronously( request, PARAMETER_LOGO );
         List<FileItem> listUploadedFileItems = _uploadHandler.getListUploadedFiles( PARAMETER_LOGO, request.getSession( ) );
-        for ( FileItem fileItem : listUploadedFileItems )
+        FileItem fileItem = listUploadedFileItems != null ? listUploadedFileItems.get(0) : null;
```

Comment:
We can keep the for loop. So if the first element of listUploadedFileItems is null, we can then look at the following elements and retrieve the first non-null element